apps/remix-ide/src/app/plugins/electron/compilerLoaderPlugin.ts

```diff
@@ -0,0 +1,69 @@
+import { ElectronPlugin } from '@remixproject/engine-electron';
+import { Plugin } from '@remixproject/engine';
+import { baseURLBin, baseURLWasm } from '@remix-project/remix-solidity'
+import axios, {AxiosResponse} from 'axios'
+import { iSolJsonBinData } from '@remix-project/remix-lib'
+
+const profile = {
+  displayName: 'compilerLoader',
+  name: 'compilerloader',
+  description: 'Loads the compiler for offline use',
+}
+
+const methods = ['getJsonBinData']
+
+export class compilerLoaderPlugin extends Plugin {
+  constructor() {
+    super(profile)
+    this.methods = methods
+  }
+
+  async getJsonBinData() {
```

Comment:
that would be helpful to get a little schema showing the architecture of Electron plugins.
I see there is `ElectronPlugin` but also `ElectronBasePluginClient`, and `getJsonBinData` is declared in this class but not in the `ElectronPlugin` one...